.vscode/settings.json

```diff
@@ -0,0 +1,18 @@
+{
+  "editor.formatOnSave": true,
+  "editor.rulers": [100],
+  "eslint.validate": [
+    "javascript",
+    "javascriptreact",
+    "typescript",
+    "typescriptreact"
+  ],
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "eslint.alwaysShowStatus": true,
+  "eslint.format.enable": true,
+  "prettier.singleQuote": true,
+  "prettier.tabWidth": 2,
+  "prettier.semi": false,
+  "prettier.trailingComma": "es5",
```

Comment:
If we remove this will it default to our `.prettierrc`?

Reply:
yes, I removed pretty much all and simplify it.